### PR TITLE
README: add a note about ansible 2.5 version

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

**Comment by alexxa** [comment]
@vrutkovs Do I understand correct that CI was triggered on this PR? If so, what do you think about updating it, so CI can ignore changes which was made in documentation, i.e. *.md files?

**Comment by vrutkovs** [comment]
@alexxa right, CI would run test on any changed file, but since we don't fully control CI we can't exclude those files.

I'll merge this PR once lgtm is received

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Install base dependencies:
 
 Requirements:
 
-- Ansible >= 2.4.3.0
+- Ansible >= 2.4.3.0, 2.5.x version is not supported
```
Comment (michaelgugino) [inline README.md]: Is this syntax valid with a comma?

**Comment by sdodson** [comment]
I'd soften the wording a bit. I think all of our code should work with ansible 2.5 but it's not what we've QE'd and support for OCP customers. I think we should be fine with Origin users using ansible 2.5 and reporting issues in github because that's the most common version they're going to have access to now that ansible-2.5 is in EPEL.

**Comment by vrutkovs** [comment]
Perhaps '2.5.x version is not yet fully supported' would sound better?

**Comment by sdodson** [comment]
Lets go with `2.5.x is not currently supported for OCP installations`

**Pull request merged**